[2022-08-06 19:39:05,419] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-27T00:00:00+00:00 [queued]>
[2022-08-06 19:39:05,440] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-27T00:00:00+00:00 [queued]>
[2022-08-06 19:39:05,440] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:39:05,440] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:39:05,441] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:39:05,464] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-27 00:00:00+00:00
[2022-08-06 19:39:05,474] {standard_task_runner.py:52} INFO - Started process 2956 to run task
[2022-08-06 19:39:05,481] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-27T00:00:00+00:00', '--job-id', '911', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpgmh3pfn_', '--error-file', '/tmp/tmp7tesezlq']
[2022-08-06 19:39:05,482] {standard_task_runner.py:80} INFO - Job 911: Subtask second_task
[2022-08-06 19:39:05,579] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-27T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:39:05,674] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-27T00:00:00+00:00
[2022-08-06 19:39:05,675] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:39:05,676] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:39:05,687] {subprocess.py:85} INFO - Output:
[2022-08-06 19:39:05,688] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:39:05,689] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:39:05,723] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220527T000000, start_date=20220806T193905, end_date=20220806T193905
[2022-08-06 19:39:05,770] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:39:05,819] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:52:56,651] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-27T00:00:00+00:00 [queued]>
[2022-08-06 21:52:56,662] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-27T00:00:00+00:00 [queued]>
[2022-08-06 21:52:56,662] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:56,662] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:52:56,662] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:56,680] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-27 00:00:00+00:00
[2022-08-06 21:52:56,688] {standard_task_runner.py:52} INFO - Started process 3136 to run task
[2022-08-06 21:52:56,691] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-27T00:00:00+00:00', '--job-id', '909', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmppg8xzt5a', '--error-file', '/tmp/tmppeq1b_i1']
[2022-08-06 21:52:56,692] {standard_task_runner.py:80} INFO - Job 909: Subtask second_task
[2022-08-06 21:52:56,769] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-27T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:52:56,876] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-27T00:00:00+00:00
[2022-08-06 21:52:56,877] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:52:56,878] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:52:56,891] {subprocess.py:85} INFO - Output:
[2022-08-06 21:52:56,893] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:52:56,893] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:52:56,928] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220527T000000, start_date=20220806T215256, end_date=20220806T215256
[2022-08-06 21:52:56,987] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:52:57,029] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:58:21,680] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-27T00:00:00+00:00 [queued]>
[2022-08-06 21:58:21,692] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-27T00:00:00+00:00 [queued]>
[2022-08-06 21:58:21,692] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:58:21,692] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:58:21,692] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:58:21,706] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-27 00:00:00+00:00
[2022-08-06 21:58:21,711] {standard_task_runner.py:52} INFO - Started process 2932 to run task
[2022-08-06 21:58:21,714] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-27T00:00:00+00:00', '--job-id', '908', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmph15ehs4o', '--error-file', '/tmp/tmpdzw92qb8']
[2022-08-06 21:58:21,715] {standard_task_runner.py:80} INFO - Job 908: Subtask second_task
[2022-08-06 21:58:21,778] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-27T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:58:21,858] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-27T00:00:00+00:00
[2022-08-06 21:58:21,859] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:58:21,859] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:58:21,871] {subprocess.py:85} INFO - Output:
[2022-08-06 21:58:21,872] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:58:21,872] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:58:21,900] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220527T000000, start_date=20220806T215821, end_date=20220806T215821
[2022-08-06 21:58:21,925] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:58:21,952] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:13:22,759] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-27T00:00:00+00:00 [queued]>
[2022-08-06 22:13:22,775] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-27T00:00:00+00:00 [queued]>
[2022-08-06 22:13:22,775] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:13:22,775] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:13:22,775] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:13:22,792] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-27 00:00:00+00:00
[2022-08-06 22:13:22,844] {standard_task_runner.py:52} INFO - Started process 2951 to run task
[2022-08-06 22:13:22,850] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-27T00:00:00+00:00', '--job-id', '912', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpmjpa5ck3', '--error-file', '/tmp/tmp3_2s0ep6']
[2022-08-06 22:13:22,850] {standard_task_runner.py:80} INFO - Job 912: Subtask second_task
[2022-08-06 22:13:22,950] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-27T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:13:23,062] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-27T00:00:00+00:00
[2022-08-06 22:13:23,063] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:13:23,063] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:13:23,080] {subprocess.py:85} INFO - Output:
[2022-08-06 22:13:23,082] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:13:23,082] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:13:23,138] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220527T000000, start_date=20220806T221322, end_date=20220806T221323
[2022-08-06 22:13:23,182] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:13:23,248] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:24:40,581] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-27T00:00:00+00:00 [queued]>
[2022-08-06 22:24:40,592] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-27T00:00:00+00:00 [queued]>
[2022-08-06 22:24:40,592] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:40,592] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:24:40,592] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:40,605] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-27 00:00:00+00:00
[2022-08-06 22:24:40,617] {standard_task_runner.py:52} INFO - Started process 3001 to run task
[2022-08-06 22:24:40,622] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-27T00:00:00+00:00', '--job-id', '913', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp7c_seova', '--error-file', '/tmp/tmp5tch018_']
[2022-08-06 22:24:40,622] {standard_task_runner.py:80} INFO - Job 913: Subtask second_task
[2022-08-06 22:24:40,710] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-27T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:24:40,799] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-27T00:00:00+00:00
[2022-08-06 22:24:40,800] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:24:40,801] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:24:40,815] {subprocess.py:85} INFO - Output:
[2022-08-06 22:24:40,817] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:24:40,817] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:24:40,865] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220527T000000, start_date=20220806T222440, end_date=20220806T222440
[2022-08-06 22:24:40,912] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:24:40,960] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
